src/electron/views/screenshot/screenshot-view.tsx

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+import * as React from 'react';
+
+export type ScreenshotViewProps = {};
+export const ScreenshotView = NamedFC<ScreenshotViewProps>('ScreenshotView', props => {
```

Comment:
It would be nice to add a `screenshot-view.test.tsx` as part of this PR with a basic "it renders like the snapshot says it should" test; we don't actually care about the snapshotted placeholder content, but having the test machinery in place helps ensure it stays tested in the future (since anyone making updates will have to at least update a snapshot)

Reply:
Ok. I added a simple test; let me know if I need to add anything to it

Reply:
LGTM!